src/codegen_function.py

```diff
@@ -101,17 +101,18 @@ def tokenize_npe_line(stmt_token, line, line_number, max_iters=64, split_token="
     :return:
     """
     def run_cpp(input_str):
-        if platform.system() == 'Windows':
-            filename = "tmp.cc"
-            tmpf = open(mode="w+", filename)
-            tmpf.write(input_str)
-            tmpf.flush()
-            tmpf.close()
-        else:
+        try:
             tmpf = tempfile.NamedTemporaryFile(mode="w+", suffix=".cc")
             tmpf.write(input_str)
             tmpf.flush()
             filename = tmpf.name
+        except:
```

Comment:
`except:` is bad, since this might throw something unrelated to what you want to catch. You should catch the type of exception you want to handle here. e.g. `except IOError:`

Reply:
removed